### DISCUSSION TITLE
Init wizard accept not just numbers

### DISCUSSION THREAD

**Discussion by Anders142536** [sileshn/ManjaroWSL2#11] [Ideas]
The init wizard that asks whether the user wants to create a new user account only accepts [1,2] as input.

I was stupid and tried to enter "yes", "y" and some other common variations of it, which obviously didn't work.

It might be more user friendly to allow "yes", "y", "no" and "n" as options as well.

**Comment by sileshn** [comment]
Yes, because its a select loop. I won't be changing that. Will think about changing the prompt msg to say you need to input 1 or 2.

**Reply by Anders142536** [reply]
yes, i think that might be sufficient. Anyways, thanks for your work! I really enjoy using it!
